src/mlbstandings/updater.py

```
@@ -51,211 +53,256 @@ class SeasonStatus(Enum):
+            )[0]
             for league in _LEAGUES
         }
         for league in _LEAGUES:
-            # TODO this would be a nice time to make sure formatting for column A is correct
+            # TODO: this would be a nice time to make sure formatting for column A is correct
             if len(column_as[league]) == 0:
                 row = self.baseballref.header_row(league)
                 self.spreadsheet.set_range(
                     f"{self._upload_sheet_name(league)}!{rc0_range_to_sheet_range(((0, 0), (0, len(row))))}",
-                    [[cast(SheetValue, 'Date')] + row])
+                    [[cast(SheetValue, "Date")] + row],
```

Ruff (0.12.2)

Consider `[cast(SheetValue, "Date"), *row]` instead of concatenation

Replace with `[cast(SheetValue, "Date"), *row]`

(RUF005)